src/modules/core/components/Button/ThreeDotsButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { MessageDescriptor } from 'react-intl';
+
+import classnames from 'classnames';
+import Button, { Props as DefaultButtonProps } from './Button';
+import Icon from '~core/Icon';
+
+import styles from './ThreeDotsButton.css';
+
+const displayName = 'ThreeDotsButton';
```

Comment:
missing `Button.` in the beginning

Reply:
I noticed that the only other button component that had the `Button.` at the beginning of the displayName was DottedAddButton. Should I update other all the other buttons displayNames for consistency?

Reply:
We resolved this conversation via Discord :)